pkgs/tools/misc/fontforge/default.nix

```diff
@@ -31,11 +31,12 @@ stdenv.mkDerivation rec {
     python freetype zlib glib libungif libpng libjpeg libtiff libxml2
   ]
     ++ lib.optionals withGTK [ gtk2 pango ]
-    ++ lib.optionals (withGTK && stdenv.isDarwin) [ Carbon Cocoa ];
+    ++ lib.optionals stdenv.isDarwin [ Carbon Cocoa ];
 
   configureFlags =
     lib.optionals (!withPython) [ "--disable-python-scripting" "--disable-python-extension" ]
-    ++ lib.optional withGTK "--enable-gtk2-use";
+    ++ lib.optional withGTK "--enable-gtk2-use"
+    ++ lib.optional stdenv.isDarwin "--without-x";
```

Comment:
I've updated the condition for "--without-x". I now see that cui doesn't start and I get some lines relevant gui:
```
TESTING: getPixmapDir:/Users/veprbl/nixpkgs/result/share/fontforge/pixmaps
TESTING: getShareDir:/Users/veprbl/nixpkgs/result/share/fontforge
TESTING: GResourceProgramDir:/Users/veprbl/nixpkgs/result/bin
trying default theme:/Users/veprbl/nixpkgs/result/share/fontforge/pixmaps/resources
fontforge.hasUserInterface!
PyFF_onAppClosing() cnt:1
Fontconfig error: Cannot load default config file
```
After this it hang ups, probably because it can't connect to xserver.